apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -337,6 +353,7 @@ defmodule Andi.InputSchemas.InputConverter do
 
   defp update_context_from_andi_step(context, "http") do
     context
+    |> cast_keys_to_atom_in_map_recursively()
```

Comment:
This was the primary cause of the malformed body bug. The message would come in with string keys, but then all the extract_steps fields expect atom keys. This would end up creating two sets of fields, both atoms and keys. Keys being the data from the message, and atoms being re-initialized empty values that get populated later (Or maybe never at all... It's hard to tell with extract_steps)